### PR TITLE
Pass context to the report to make it easier to customize the notebooks

### PR DESCRIPTION
**Pull request by aloukina**
The context (rsmtool or rsmeval) is passed as an environment variable to the header to make it easier to adjust the notebooks depending on whether they are used for rsmeval or rsmtool.

### CODE REVIEW

File: doc/new_notebooks.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ When writing such notebooks, some or all of the python variables below will be a
 
 - `description`: The description string from the config file. 
 
+- `context` : whether the report is generated by `rsmtool` or `rsmeval`.
```
Comment (desilinguist) [inline doc/new_notebooks.md]: whether the report is BEING generated by rsmtool or rsmeval?

**Comment by desilinguist** [comment]
Just to be clear. This is just the set up right? You haven't actually used the context variable in the notebooks yet?

**Comment by aloukina** [comment]
I have - in one of the special notebooks

**Comment by desilinguist** [comment]
I see you are assigning the environment variable `CONTEXT` to a python variable but it's not actually being used to do anything, right? Am I missing something?

**Comment by desilinguist** [comment]
Looks good to me.

**Pull request merged**